docker-compose-production.yml

```diff
@@ -0,0 +1,40 @@
+app:
+  build: .
+web:
+  extends:
+    service: app
+  command: gunicorn -b 0.0.0.0:8000 --reload measurements.wsgi --workers 4 --timeout 120
+  ports:
+    - "8000"
+  links:
+    - redis
+    - rabbitmq
+https-portal:
+  image: steveltn/https-portal:1.0.0
+  ports:
+    - "80:80"
+    - "443:443"
+  links:
+    - web
```

Comment:
It seems like this is the only thing that is being added to the production docker-compose file. Can you not just add this to `config/production.yml`?